README.md

```diff
@@ -30,13 +30,20 @@ There is currently one public instance running at
 
 # Development / Running
 
-```
-npm install
-mkdir build
-# Dev: watchify -t [ babelify --presets [ react es2015 ] ] src/index.js -o build/app.js
-browserify -t [ babelify --presets [ react es2015 ] ] src/index.js -o build/app.js
-go build
-./leapchat
-```
+To install and build static assets:
+
+``` $ npm install ```
+
+``` $ npm run build ```
```

Comment:
@jimmcgaw I made these tweaks, and now everything works great for me: https://github.com/cryptag/leapchat/pull/41/commits/921808cf455b664f550463019fc735bc390e675d